### PR TITLE
chore: upgrade deps

### PR DESCRIPTION
**Pull request by webfansplz**

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -87,6 +87,9 @@ jobs:
       - name: Install
         run: pnpm i
 
+      - name: Dev
+        run: pnpm run dev
```
Comment (posva) [inline .github/workflows/test.yml:91]: I don't think this one should be included in the test workflow as it hangs

File: tsconfig.json [hunk 2]
```diff
@@ -1,6 +1,5 @@
 {
   "compilerOptions": {
-    "rootDir": ".",
```
Comment (posva) [inline tsconfig.json:3]: was this breaking anything? 🤔

File: .github/workflows/test.yml (hunk 1 above)
Comment (webfansplz) [reply .github/workflows/test.yml:91]: image
image
Related to this. We got error while run the `test` command. Maybe there's a better way to handle it ?

Comment (webfansplz) [reply .github/workflows/test.yml:91]: > image
> 
> image
> 
> Related to this. We got error while run the ``test`` command. Maybe there's a better way to handle it ?

File: tsconfig.json (hunk 2 above)
Comment (posva) [reply tsconfig.json:3]: that's weird as the tsconfig is at the root of all packages. Maybe we should add another local tsconfig that extends this one

**Review by posva — APPROVED** [review]
I renamed the `dev` script at the root and in sub packages as `stub` while keeping the other scripts named `dev` to better differentiate them and added a small note about it to the contributing.md file.
Thanks for this!

**Pull request merged**